app/backend/src/Application.ts

```diff
@@ -41,6 +44,17 @@ export class Application {
       const api: Api = new Api(config.api.port, logger, database);
       await api.listen();
 
+      // Make axios proxy to work with ethers.js
+      const env = getEnv();
+      const httpProxy = env.optionalString("HTTPS_PROXY");
```

Comment:
Node uses this variable to proxy all HTTPS requests through the specified endpoint. For whatever reason it worked for regular calls, like coincap/coingecko, but not for ones made by ethers.js. This part fixes it.

Can move to any other place if there are any suggestions.